### PR TITLE
Modify rawards apy

### PR DESCRIPTION
**Pull request by robertg93**

### CODE REVIEW

File: tests/testUtils.ts [hunk 1]
```diff
@@ -371,4 +371,520 @@ export const createTickArray = (size: number) => {
...
+      liquidityChange: { v: '19068164041883237' },
+      liquidityGross: { v: '19068164041883237' },
+      sqrtPrice: { v: '2000436350662000000000000' },
+      feeGrowthOutsideX: { v: '0' },
+      feeGrowthOutsideY: { v: '0' },
+      secondsPerLiquidityOutside: { v: '0' },
+      pool: 'FwiuNR91xfiUvWiBu4gieK4SFmh9qjMhYS9ebyYJ8PGj'
+    }
+  ],
+  weeklyData: {
```
Comment (wojciech-cichocki) [inline tests/testUtils.ts:873]: Do we need `weeklyData`?

File: tests/testUtils.ts [hunk 2]
```diff
@@ -371,4 +371,520 @@ export const createTickArray = (size: number) => {
...
+      index: -13860,
+      sign: true,
+      bump: 251,
+      liquidityChange: { v: '22964554844140308' },
+      liquidityGross: { v: '22964554844140308' },
+      sqrtPrice: { v: '500090922499000000000000' },
+      feeGrowthOutsideX: { v: '124251168448391763' },
+      feeGrowthOutsideY: { v: '146254149906104013' },
+      secondsPerLiquidityOutside: { v: '264697' },
+      pool: 'FwiuNR91xfiUvWiBu4gieK4SFmh9qjMhYS9ebyYJ8PGj'
```
Comment (wojciech-cichocki) [inline tests/testUtils.ts:412]: pool filed seems to be redundant. Just mock data, so it isn't problem

File: tests/math.spec.ts [hunk 3]
```diff
@@ -1899,82 +1899,84 @@ describe('Math', () => {
...
-      // avg(dailyFactor) = 0.002145966...
-      // APY = 118.679796944...
-      const poolApy = poolAPY(paramsApy)
-      assert.ok(poolApy.apy > 118 && poolApy.apy < 119)
-      assert.ok(poolApy.weeklyRange.length === 7)
-      assert.ok(poolApy.weeklyRange[0].tickLower === -700)
-      assert.ok(poolApy.weeklyRange[0].tickUpper === 2000)
-    })
-  })
+  // describe('pool APY tests', () => {
```
Comment (wojciech-cichocki) [inline tests/math.spec.ts:1902]: should it be commented?

**Review by wojciech-cichocki — APPROVED** [review]

File: tests/testUtils.ts (hunk 1 above)
Comment (robertg93) [reply tests/testUtils.ts:873]: removed

File: tests/math.spec.ts (hunk 3 above)
Comment (robertg93) [reply tests/math.spec.ts:1902]: restored

File: tests/testUtils.ts (hunk 2 above)
Comment (robertg93) [reply tests/testUtils.ts:412]: this field has to be there because it's tick interface

**Pull request merged**